docs/tools/CLI/cli-collaborate.md

```diff
@@ -101,6 +101,35 @@ $ mbed export -i uvision -m K64F
 
 Mbed CLI creates a `.uvprojx` file in the root project directory. You can open the project file with uVision.
 
+#### Serial terminal
+
+You can open a serial terminal to the COM port of a connected Mbed target (usually board) using the `mbed sterm` command. If no COM port is specified, Mbed CLI will attempt to detect the connected Mbed targets and their COM ports.
```

Comment:
`COM` port is a windows-only term. They're called `tty...` on linux and OSX. Could we replace COM with serial?